change/@fluentui-contrib-react-keytips-6c388a16-92f8-4352-a0e6-37f7bdd0eb14.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

this is wrong - we operate in zero major semver policy

- major - not allowed
- minor - breaking changes
- patch - feature/fix

```suggestion
  "type": "patch",
```